source/gui/inputGestures.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: UTF-8 -*-
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2020 NV Access Limited
+# Copyright (C) 2013-2020 NV Access Limited, James Teh, Zahari Yurukov, Joseph Lee, Reef Turner,

```

Comment:
When working on #13294 I've verified that the copyright header herer is accurate, though it seems we don't include names of NV Access employees.
```suggestion
# Copyright (C) 2013-2020 NV Access Limited, Zahari Yurukov, Joseph Lee,
```